exporter/redis.go

```diff
@@ -859,6 +896,12 @@ func (e *Exporter) scrapeRedisHost(scrapes chan<- scrapeResult, addr string, idx
 
 	e.extractInfoMetrics(infoAll, addr, e.redis.Aliases[idx], scrapes, dbCount)
 
+	if serverInfo, err := redis.Strings(doRedisCmd(c, "SERVER")); err == nil {
+		e.extractTile38Metrics(serverInfo, addr, e.redis.Aliases[idx], scrapes)
+	} else {
+		log.Errorf("Redis SERVER err: %s", err)
```

Comment:
Maybe leave a comment before line 899 that this will succeed for Tile38 servers only and then make this a `Debugf()`, otherwise it'll spam the logs for regular redis instances.

Reply:
@oliver006 Really thanks for your time.
`http_transport` exposes a boolean that indicates HTTP transport is using or not.
About debugging it is my mistake and I will switch it to debug level log.
And what do you think about tests? Are you agree with adding tile38 into circle-ci and tests?

Reply:
Ah ok, thanks for explaining. Your changes look good but yeah, adding a test for the newly introduced parsing function is probably a good idea.

Reply:
Ok, I will add tests for tile38.